.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "workbench.preferredDarkColorTheme": "Visual Studio Light",
+    "workbench.preferredHighContrastColorTheme": "Default High Contrast Light",
+    "window.autoDetectColorScheme": true
+}
```

Comment:
[OPTIONAL] since this is your own VSCode setting. it doesn't necessarily need to be committed into the repo. so having this in the .gitignore file may be a good choice 👍